climpred/comparisons.py

```diff
@@ -54,7 +53,7 @@ def _stack_to_supervector(ds, new_dim='svd', stacked_dims=('init', 'member')):
 # PERFECT-MODEL COMPARISONS
 # based on supervector approach
 # --------------------------------------------#
-def _m2m(ds, supervector_dim='svd'):
+def _m2m(ds, supervector_dim='svd', stack=True):
     """
     Create two supervectors to compare all members to all others in turn.
 
```

Comment:
Add description of `stack` here for developers.

Reply:
added in args

Reply:
@aaronspring , maybe this should be changed to `stack_dims` everywhere to be more explicit.

Reply:
`stacked_dims` is already taken in `_stack_to_supervector`.
now `stack` changes to `stack_dims`